backend/migrations/00040_alter_programs.sql

```diff
@@ -0,0 +1,90 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE public.programs ADD COLUMN program_owner VARCHAR(255);
+ALTER TABLE public.programs ADD COLUMN funding_type VARCHAR(50);
```

Comment:
It sounds like Program Type, Funding Type, and Credit Type are going to be multi-select. 
I think Program Type can mean one of many categories (tags?).
How do we want to handle multiple values for credit type and funding type? Does it need to be with many-to-many relation? Or should we have a Programs_CreditTypes (program_id, credit_type) and a Programs_FundingTypes (program_id, funding_type) table and use enums for the values?